Controller/TemplatingExceptionController.php

```diff
@@ -25,7 +24,7 @@ public function __construct(
         ViewHandlerInterface $viewHandler,
         ExceptionValueMap $exceptionCodes,
         $showException,
-        EngineInterface $templating
+        $templating
```

Comment:
I think we should add a check which ensure that `$templating` is either an implementation of the `EngineInterface` or an instance of `Twig\Environment` and throw a `TypeError` otherwise.

Reply:
I added this in 3a18517434573e422ebac63406aa408b73229690, I also wrote a test but saw there weren't any really for stuff like that; if you want it, let me know and I can add it (see it here: https://gist.github.com/sarcher/300c605623ab131d7b78c821a99b0e34)